qutebrowser/components/misccommands.py

```diff
@@ -339,16 +339,23 @@ def debug_webaction(tab: apitypes.Tab, action: str, count: int = 1) -> None:
 
 @cmdutils.register()
 @cmdutils.argument('tab', value=cmdutils.Value.count_tab)
-def tab_mute(tab: Optional[apitypes.Tab]) -> None:
+@cmdutils.argument('to_set', choices=['true', 'false'])
+def tab_mute(tab: Optional[apitypes.Tab], to_set: str = None) -> None:
     """Mute/Unmute the current/[count]th tab.
 
     Args:
         count: The tab index to mute or unmute, or None
+        to_set: The mute setting to force: 'true' 'false' or 'None
```

Comment:
I don't think we need to specify "None" here, that's the same as not passing the argument and I don't think we note the optionality of arguments in docstrings.

Reply:
This is shown to the user in the `:help` - from the users perspective, `None` makes no sense here indeed. But it should perhaps point out "If not given, the mute state gets toggled." or something along those lines.

Reply:
Didn't know that's where the ``:help`` information comes from, I'll update this.